.github/workflows/unit-tests.yml

```diff
@@ -5,6 +5,14 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+    paths:
+      - 'distributions/**'
+      - 'llama_stack/**'
+      - 'tests/**'
```

Comment:
do you want to distinguish here between the two types? (trigger on unit-tests only here; and on integration-tests only in the other file)?

Reply:
very good point!